### ISSUE TITLE
NullReferenceException when pressing toggleGroupsButton

### ISSUE THREAD

**Issue by mercutiodesign** [X2CommunityCore/xcom2-launcher#12]
Fresh download of XCOM2ModLauncher v1.1.2, after adding a mod to a group I wanted to see them but pressing the button leads to the following exception:

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at XCOM2Launcher.Forms.MainForm.modlist_toggleGroupsButton_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

[full-exception-text.txt](https://github.com/X2CommunityCore/xcom2-launcher/files/1144475/full-exception-text.txt)

**Comment by RevZero** [comment]
This happens when the mod list is empty. 

In that case, the OLV doesn't create any goups and OLVGroups is NULL. This can be easily fixed in the next version.

**Comment by RevZero** [comment]
Has been fixed in 970a7adc0861d7181da3da39482b74a4975b80e9.
